tests/dft-grad-lr2/input.dat

```diff
@@ -0,0 +1,44 @@
+#! Tests CAM gradients with and without XC pieces to narrow grid error
+
+func_reference = psi4.Matrix.from_list([                         #TEST
+       [   0.000000000000,  -0.000000000000,  -0.052360092488],  #TEST
+       [  -0.000000000000,   0.000000000000,   0.052360092488]]) #TEST
+cam_b3lyp_reference = psi4.Matrix.from_list([                    #TEST
+       [   0.000000000000,  -0.000000000000,  -0.109138077779],  #TEST
+       [   0.000000000000,  -0.000000000000,   0.109137316634]]) #TEST
+
+molecule {
+    H  0.000  0.000  0.000
+    F  0.000  0.000  1.100
+}
+
+
+set {
+    scf_type              df
+    basis                 cc-pvdz
+    dft_radial_points     120
+    dft_spherical_points  434
+    e_convergence         12
+    d_convergence         12
+    points 5
+}
+
+func = {
+    "name": "random",
+    "x_hf": {
+        "alpha": 0.6,
+        "beta": 0.9,
+        "omega": 0.2
+    },
```

Comment:
Oh yes, I see why you modified the functional builder. It would perhaps be cleaner to leave the correlation check in place and add `"c_functionals": {}` here instead?

Reply:
+1, fixed.